test/e2e/infinispan/upgrade_operand_test.go

```diff
@@ -114,8 +124,12 @@ func TestOperandCVERollingUpgrade(t *testing.T) {
 // the currently installed operand, result in a GracefulShutdown upgrade
 func TestOperandCVEGracefulShutdown(t *testing.T) {
 	defer testKube.CleanNamespaceAndLogOnPanic(t, tutils.Namespace)
+	versionManager := tutils.VersionManager()
+
+	if !versionManager.Latest().CVE {
+		t.Skip("Latest release is non-cve, skipping test")
+	}
```

Comment:
If we're going to skip the test, then we can remove the `if !cveOperand.CVE {` check on L146-150.

Reply:
Removed